docs/faq.md

```diff
@@ -77,11 +77,15 @@ good to them.
 
 ## I'd like to get my fuzzer evaluated on the free FuzzBench service, but I don't want the results to be public yet.
 
-Please reach out to us at fuzzbench@google.com.
+Please reach out to us at fuzzbench@google.com. If we agree to benchmark your
+fuzzer, please send us a patch file for fuzzbench and please test your patch
+using `make build-$FUZZER-all`.
 
 ## I'm working on a new tool, but I'm not ready to make my code public yet. Can I use the FuzzBench service?
 
-Yes, please reach out to us at fuzzbench@google.com.
+Yes, please reach out to us at fuzzbench@google.com. If we agree to benchmark
+your tool, please send us a patch file for fuzzbench and please test your
+patch using `make build-$TOOL-all`.
```

Comment:
Do you think `make build-$TOOL-all` is too burdensome? It will take a long time. It might be better just to ask for `make build test-run-$TOOL-$OSS_FUZZ_BENCHMARK test-run-$TOOL-$STANDARD_BENCHMARK`

Reply:
Yes right, i think can ask them to try against one benchmark OR one oss-fuzz,regular benchmark. wonder if example benchmark can go stale, so can ask them to try against 1-2 benchmarks ?

Reply:
We can also say: try at at least 1-2 benchmark, ideally all.

Reply:
ah right, all will be a nightmare. 1-2 seems better ?

Reply:
OK updated the docs to say this.